api/v1alpha1/rpaasinstance_types.go

```diff
@@ -134,6 +134,12 @@ type RpaasInstanceSpec struct {
 	// +optional
 	// +kubebuilder:default:=false
 	Suspend *bool `json:"suspend,omitempty"`
+
+	// Shutdown flag tells whether controller should scale down Nginx instances.
+	// Any assosciated HorizontalPodAutoscaler is remove/created when this flag is toggled.
+	// +optional
+	// +kubebuilder:default:=false
+	Shutdown *bool `json:"shutdown,omitempty"`
```

Comment:
@gvicentin I think that just use `bool` will be fine:
when you use `*bool` you have three possible values [null, &false, &true]

when you use `bool` you have just two possible values [false, true]
this is fine and reduce liability of dealing with pointers.

Reply:
Agreed, thanks @wpjunior!